### ISSUE TITLE
Installation Issue: Wisdom not being made correctly (for me)

### ISSUE THREAD

**Issue by afinemax** [scottransom/presto#199]
Greetings!

I am attempting to install Presto, following the instructions. 

I have cloned the repo, and have ran:
`meson compile -C build
meson install -C build`

And everything looks generally ok? I can run one of the test scripts, but then I get a message telling me that wisdom wasn't made correctly. I recall  running `meson compile -C build' more then once and seeing it make the wisdom file, but I think I killed the terminal session before it was done. 

`Warning:  Couldn't open '/home/afinemax/afinemax/khazad-dum/research/astron_2024/presto_and_tut/presto/lib/fftw_wisdom.txt'
          You should run 'makewisdom'.  See $PRESTO/INSTALL`
          
Another error says: `sh: 1: tempo: not found`, which is odd because I think I installed tempo and added it to my `$PATH'. 

I haven't looked at the MESON logs but maybe something there might be of help? I have tried rerunning the MESON build commands but I just get a message saying that it is already built.  I am not sure how to proceed. 

the full message after running `test_presto_python.py' is here:

`(astron_research_2024) afinemax@khazad-dum:~/afinemax/khazad-dum/research/astron_2024/presto_and_tut/presto$ 
python tests/test_presto_python.py
Warning:  Couldn't open '/home/afinemax/afinemax/khazad-dum/research/astron_2024/presto_and_tut/presto/lib/fftw_wisdom.txt'
          You should run 'makewisdom'.  See $PRESTO/INSTALL.
Testing FFT stuff... success
Testing FFTW call... success
Warning:  Couldn't open '/home/afinemax/afinemax/khazad-dum/research/astron_2024/presto_and_tut/presto/lib/fftw_wisdom.txt'
          You should run 'makewisdom'.  See $PRESTO/INSTALL.
Testing tablesixstepfft call... success
Testing reading infiles... success
Testing writing infiles... success
Note: If the following fails, edit $PRESTO/meson.build and remove
        '-DUSE_FFTW_MALLOC' from the `_global_c_args` definition, and re-build.
Testing allocation and freeing of memory... success
Testing psrparams and orbitparams stuff... success
Testing spectralpower and spectralphase... success
Testing vector shifting / rotation... success
Testing orbit integration stuff... success
Testing Fourier response generation... success
Testing angle functions... success
sh: 1: tempo: not found

Error in chkfopen(): No such file or directory
   path = 'resid2.tmp'
(astron_research_2024) afinemax@khazad-dum:~/afinemax/khazad-dum/research/astron_2024/presto_and_tut/presto$ 
python examplescripts/ffdot_example.py
Warning:  Couldn't open '/home/afinemax/afinemax/khazad-dum/research/astron_2024/presto_and_tut/presto/lib/fftw_wisdom.txt'
          You should run 'makewisdom'.  See $PRESTO/INSTALL.
Fraction of recovered signal power = 0.516949
Time for rz: 0.0017822630006776308
4096.0 4095.196532252572 10.0 9.586850057632322 67108864.0 20466245.978796482
Time for rzw: 0.6664392479997332
4096.0 4096.0 10.0 9.91955907509756 -40.0 -40.637606213020426 67108864.0 67549048.0
 Type <RETURN> for next page: 

`

**Comment by scottransom** [comment]
Hi Maxwell:  Sorry for the delay responding.  I was on vacation last week!

Two quick comments:  Several of these issues are definitely happening because TEMPO is not being found.  To check to make sure that it is installed and in your PATH correctly, do you get a good result for `which tempo`?  And if so, can you run that executable and get the text help info from TEMPO?

As for the wisdom file, you definitely need to let it complete before you copy it to the appropriate library directory.  I have added a sentence about that in the INSTALL.md file:  "If you want to run makewisdom for slightly faster FFT calls, it is located in $PRESTO/build/src. Just run it from there, and then copy or move the resulting fftw_wisdom.txt file to $PRESTO/lib."

Let me know if this fixes things.

**Comment by afinemax** [comment]
Yes, it was certainly a tempo issue.  I have a working installation now + the docker version
